File: algorithms/designer-pdf-viewer/designerPdfViewer.php
<?php
function designerPdfViewer($h, $word) {
  $alphabetic = "abcdefghijklmnopqrstuvwxyz";
  $heights = [];

  foreach ($word as $letter) {
    $i = strpos($alphabetic, $letter);
    $heights[] = $h[$i];
  }
  $highlightedArea = max($heights) * count($word);

  return $highlightedArea;
}
?>

File: algorithms/designer-pdf-viewer/designer_pdf_viewer.py
def designerPdfViewer(h:list, word:str) -> int:
  alphabetic = list("abcdefghijklmnopqrstuvwxyz")
  heights = list()
  for letter in word:
    i = alphabetic.index(letter)
    heights.append(h[i])

  highlighted_area = max(heights) * 1 * len(word)

  return highlighted_area 